scripts/netContain/README.md

```diff
@@ -1,42 +1,62 @@
-## Introduction 
```

Comment:
Removes redundant whitespace. This pollutes diffs when added/removed. It's better to keep an eye on it and remove it.